pkg/spdk/server.go

```diff
@@ -296,11 +316,16 @@ func (s *Server) ReplicaCreate(ctx context.Context, req *spdkrpc.ReplicaCreateRe
 	}
 
 	s.Lock()
+	spdkClient := s.spdkClient
 	if _, ok := s.replicaMap[req.Name]; !ok {
+		ready, err := s.checkLvsReadiness(req.LvsUuid, req.LvsName)
+		if err != nil || !ready {
+			s.Unlock()
```

Comment:
Can we just move `s.Unlock()` to just next to `s.Lock()`, and then remove the following original `s.Unlock()`?

Reply:
Addressed. Thank you.